### DISCUSSION TITLE
Download RAW photos (DGN files)

### DISCUSSION THREAD

**Discussion by pjahn** [icloud-photos-downloader/icloud_photos_downloader#629] [Q&A]
I am using the latest version of icloudpd (1.13). I sometimes take RAW photos with the iPhone. icloupd tells me te following when I try to download photos from icloud taken with RAW option: Skipping IMG_0250.DNG, only downloading photos.

Is there a way to include DGN files in the download?

Thank you for help.

**Comment by AndreyNikiforov** [comment]
There is no way to download these type of files, currently. Thanks for suggesting an improvement.

**Comment by AndreyNikiforov** [comment]
[RAW files are supported since 1.19.0](https://icloud-photos-downloader.github.io/icloud_photos_downloader/raw.html)
